src/libsync/propagateupload.h

```diff
@@ -220,5 +259,38 @@ private slots:
     void abortWithError(SyncFileItem::Status status, const QString &error);
 };
 
+/**
+ * @brief The PropagateBundleQNAM class
+ * @ingroup libsync
+ */
+class PropagateBundleQNAM : public PropagateItemJob {
```

Comment:
don't add QNAM in the job t-name.  This was just to distinguish the older joobs from the legacy neon one.  But today, no reason to keep QNAM in the name.